src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -286,6 +286,18 @@ public override void VisitMarkupTagHelperAttribute(MarkupTagHelperAttributeSynta
             AddSemanticRange(node.ValueSuffix, RazorSemanticTokensLegend.MarkupAttributeQuote);
         }
 
+        public override void VisitMarkupTagHelperAttributeValue(MarkupTagHelperAttributeValueSyntax node)
+        {
+            if(node.Children.First().Kind == SyntaxKind.MarkupTextLiteral)
+            {
+                AddSemanticRange(node, RazorSemanticTokensLegend.MarkupAttributeQuote);
```

Comment:
I forget, is it a workaround that we're indicating its a "markupattributequote"?

Reply:
Yeah, the VS Tagger had a small mistake (my fault) which caused `markupAttributeValue` to not be colorized. I've submitted a fix, but it hasn't made it into VS main yet. In the mean time MarkupAttributeQuote maps to the same classification so I've been using it to keep the experience good.